websocket.go

```diff
@@ -318,7 +319,9 @@ func (c *Client) newConnIfNotExist(dst *network.ServerIdentity, path string) (*w
 	// TODO we are opening a new connection for every new path?
```

Comment:
Not related to the PR but this TODO can go away imho. Connections are tight to their end points and it's not designed to be reused.